core/src/main/scala/tech/beshu/ror/accesscontrol/factory/RawRorConfigBasedCoreFactory.scala

```diff
@@ -55,6 +55,7 @@ import tech.beshu.ror.utils.yaml.YamlOps
 import java.time.Clock
 
 final case class CoreSettings(aclEngine: AccessControl,
```

Comment:
IMO we should:
* change the name of this class to `Core` (Settings can be misleading when we have RorConfig in it)
* move auditingSettings to RorConfig class
* rename `AclCreationError` to `CoreCreationError`